src/models/scryfall-card.ts

```diff
@@ -33,6 +33,9 @@ export type DefaultScryfallCard = {
   set: string;
   image_uris: ScryfallImageUris;
   card_faces?: never;
+  cmc: number;
```

Comment:
This is duplicated for now and I notice this exists https://github.com/scryfall/api-types, checking  if this can be in another PR or in this one?